Library/Formula/speech-tools.rb

```diff
@@ -0,0 +1,23 @@
+class SpeechTools < Formula
+  homepage "http://festvox.org/docs/speech_tools-2.4.0/"
+  url "http://festvox.org/packed/festival/2.4/speech_tools-2.4-release.tar.gz"
+  sha1 "5b0ebd39bb7afa33e4093724a2123bdc62a6aebc"
+
+  def install
+    ENV.deparallelize
+    system "./configure"
+    system "make"
```

Comment:
If you're expected to add `ESTDIR/bin` into your PATH also why isn't this installing the contents of a `bin` directory?